.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-cpython:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        name: [
+          "ubuntu-py37",
+          "ubuntu-py36",
```

Comment:
No more 2.7?  oh, so sad.  But should that be reflected in `setup.py`?

Reply:
done at https://github.com/gramaziokohler/roslibpy/pull/72/commits/f4d9cc16181c1678f403d53381e7446fcde71534